cogs/events.py

```diff
@@ -78,7 +76,15 @@ async def on_ready(self):
         )
 
         # Indicate that the bot has successfully booted up
-        print(f"Ready: {self.bot.user} | Servers: {len(self.bot.guilds)}")
+        print(f"\nLogged in as: {self.bot.user.name} - {self.bot.user.id}\n"
+        f"Version: {discord.__version__}")
+        member_count = 0
+        guild_string = ""
+        for g in self.bot.guilds:
+            guild_string += f"-> {g.name} - ID: {g.id} - Members: {g.member_count}\n"
+            member_count += g.member_count
+        print(
+            f"Active on {len(self.bot.guilds)} guilds:\n{guild_string}")
```

Comment:
Instead of doiung multiple prints, possibly do one pring with multiple variables, and newline stuff can be done like this
```py
print("\n".join([
   "This is the first line",
   "Second line, but automatically new-lined",
   ...
]))

Reply:
Also... you can use a list comprehension with str.join() for the guild_string:
```python
formatted_guilds = [
    f"Name: {g.name} - ID: {g.id} - Members: {g.member_count}"
    for g in self.bot.guilds
]
guild_string = "\n".join(formatted_guilds)
```
And for a total unique user count you can do something like: `len(set(self.bot.get_all_members()))` or `sum(g.member_count for g in bot.guilds)` but that wouldn't be unique count.

Reply:
On it